build.gradle.kts

```diff
@@ -53,3 +71,15 @@ publishing {
         }
     }
 }
+
+tasks.withType<KotlinCompile> {
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8.toString()
```

Comment:
Can we have `JavaVersion` defined _once_ and used across the build file(s), please?

Reply:
I mean, we keep saying "We use Java 8, we use Java 8, we use Java 8" in this code. Can we DRY, instead?